### PR TITLE
enh(cpp) add `pair`, `make_pair`, `priority_queue` as built-ins

### PR DESCRIPTION
**Pull request by Linhk1606**
update a little: use `priority_queue` `pair` `make_pair` as built-ins of c++

### CODE REVIEW

**Comment by yyyc514** [comment]
Update change log please.

**Comment by Linhk1606** [comment]
> Update change log please.

OK.

**Comment by yyyc514** [comment]
Just waiting on a changelog on this and then I can merge.

**Comment by Linhk1606** [comment]
...Excuse me, but where is the changelog? :smile:

**Comment by yyyc514** [comment]
`CHANGES.md` in root of project.

File: CHANGES.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Language Improvements:
 - enh(typescript/javascript/coffeescript/livescript) derive ECMAscript keywords from a common foudation (#2518) [Josh Goebel][]
 - enh(typescript) add setInterval, setTimeout, clearInterval, clearTimeout (#2514) [Josh Goebel][]
 - enh(javascript) add setInterval, setTimeout, clearInterval, clearTimeout (#2514) [Vania Kucher][]
+- enh(cpp) add pair, make_pair, priority_queue as built-ins (#2538) [Linhk1606][]
```
Comment (yyyc514) [inline CHANGES.md]: You need to add a reference on line ~36 to tell it where your name should link to.

File: CHANGES.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Language Improvements:
 - enh(typescript/javascript/coffeescript/livescript) derive ECMAscript keywords from a common foudation (#2518) [Josh Goebel][]
 - enh(typescript) add setInterval, setTimeout, clearInterval, clearTimeout (#2514) [Josh Goebel][]
 - enh(javascript) add setInterval, setTimeout, clearInterval, clearTimeout (#2514) [Vania Kucher][]
+- enh(cpp) add pair, make_pair, priority_queue as built-ins (#2538) [Hankun Lin][]
```
Comment (yyyc514) [inline CHANGES.md:27]:
```suggestion
- enh(cpp) add `pair`, `make_pair`, `priority_queue` as built-ins (#2538) [Hankun Lin][]
```

**Comment by yyyc514** [comment]
Thank you so much for your contribution!

**Pull request merged**